lib/matplotlib/tests/test_pyplot.py

```diff
@@ -367,3 +368,26 @@ def test_set_current_axes_on_subfigure():
     assert plt.gca() != ax
     plt.sca(ax)
     assert plt.gca() == ax
+
+
+def test_pylab_integration():
+    pytest.importorskip("IPython")
+    subprocess.run(
+        [
+            sys.executable,
+            "-m",
+            "IPython",
+            "--pylab",
+            "-c",
+            ";".join((
+                "import matplotlib.pyplot as plt",
+                "assert plt._REPL_DISPLAYHOOK == plt._ReplDisplayHook.IPYTHON",
+            )),
+        ],
+        env={**os.environ, "SOURCE_DATE_EPOCH": "0"},
```

Comment:
You don't really need this, nor the stdout/stderr capture (pytest does it for you), nor universal_newlines, I think?